### PR TITLE
Grammar fix

### PR DESCRIPTION
**Pull request by emmaodia**
Replaces  #13461.

### CODE REVIEW

File: docs/types/value-types.rst [hunk 1]
```diff
@@ -221,7 +221,7 @@ Operators:
 
 .. warning::
     If you convert a type that uses a larger byte size to an ``address``, for example ``bytes32``, then the ``address`` is truncated.
-    To reduce conversion ambiguity version 0.4.24 and higher of the compiler force you make the truncation explicit in the conversion.
+    To reduce conversion ambiguity version 0.4.24 and higher versions of the Solidity compiler force you to make the truncation explicit in the conversion.
```
Comment (Marenz) [inline docs/types/value-types.rst]:
```suggestion
    To reduce conversion ambiguity, starting with version 0.4.24 the compiler will force you to make the truncation explicit in the conversion.
```

I don't think we need to explicitly mention "solidity" in the solidity documentation :)

**Review by Marenz — CHANGES_REQUESTED** [review]

File: docs/types/value-types.rst [hunk 2]
```diff
@@ -221,7 +221,7 @@ Operators:
 
 .. warning::
     If you convert a type that uses a larger byte size to an ``address``, for example ``bytes32``, then the ``address`` is truncated.
-    To reduce conversion ambiguity version 0.4.24 and higher of the compiler force you make the truncation explicit in the conversion.
+    To reduce conversion ambiguity, starting with version 0.4.24 the compiler will force you to make the truncation explicit in the conversion.
```
Comment (matheusaaguiar) [inline docs/types/value-types.rst]:
```suggestion
    To reduce conversion ambiguity, starting with version 0.4.24, the compiler will force you to make the truncation explicit in the conversion.
```
Don't want to be annoying but it seems to me that a comma is needed here.

**Review by matheusaaguiar — COMMENTED** [review]
I noticed the commit message has misspelled the word `Grammar`... guess it would be nice to amend it.

**Review by matheusaaguiar — DISMISSED** [review]
LGTM. 
Please, squash commits.

**Review by Marenz — APPROVED** [review]

**Comment by nikola-matic** [comment]
@emmaodia can you please rebase against `develop`, the failing steps should now be fixed (not all, but at least the required ones).

**Comment by leonardoalt** [comment]
@nikola-matic @Marenz @matheusaaguiar if the author is unresponsive you can also just rebase and push directly into their branch/fork

**Comment by leonardoalt** [comment]
re: merge commit which should be removed and squashing

**Comment by nikola-matic** [comment]
> @nikola-matic @Marenz @matheusaaguiar if the author is unresponsive you can also just rebase and push directly into their branch/fork

Author pushed 5 days ago after receiving Mathias' approval, so let's wait for a day or two.

**Review by Marenz — CHANGES_REQUESTED** [review]
PR content is good, just waiting for a rebase + fixup/amending of commits into one.

**Comment by emmaodia** [comment]
Thank you @ekpyron @nikola-matic @Marenz @leonardoalt I was away and I missed the messages. Thanks again.

**Pull request merged**